back-end/server/index.js

```diff
@@ -56,9 +56,15 @@ const createApp = () => {
   app.use(passport.initialize())
   app.use(passport.session())
 
-  // auth and api routes
+  // auth, gate, and api routes
+
   app.use('/auth', require('./auth'))
-  app.use('/api', require('./api'))
+
+  const gateway = (req, res, next) => {
+    req.user ? next() : res.sendStatus(403)
```

Comment:
Instead of doing res.sendStatus(403), you can do next(createError('Not Authorized', 401)), it will show a proper message when you try it without logging in. Also, this is 401 error, not a 403.